### DISCUSSION TITLE
New label for "members only" issues

### DISCUSSION THREAD

**Discussion by rndquu** [ubiquity/ubiquibot#397] [Ideas]
Sometimes there are issues that should be handled **only** by a project's core team because:
1. Core team members deliver quality code
2. Core team must be sure that a feature is implemented in the best way possible

Examples of such critical issues:
- https://github.com/ubiquity/ubiquity-dollar/issues/636
- https://github.com/ubiquity/ubiquibot/issues/176

So we could add a new tag, like `Core` or `Members only` which would make sure that an issue could be assigned **only to an organisation member**.

**Comment by 0x4007** [comment]
I dislike clouding label space because other projects likely have their own label systems that we need to compete with for visual/UI real estate. The more labels we have as dependencies, the less attractive it could be to use our system for this reason.

This can be solved by explicitly delegating tasks to the core team, either via direct communication (e.g. phone call or private message) or assigning on GitHub, but most likely both. 

The simple workaround could be that we don't consider those as bounties until AFTER a core team member picks it up. So we remove the `Time` or `Priority` label, making it impossible for a bounty hunter to pick up. Once a core team member is assigned (or assigns themselves using the GitHub UI) then we can add the missing label to make it eligible for the bounty.

**Reply by rndquu** [reply]
> The simple workaround could be that we don't consider those as bounties until AFTER a core team member picks it up. So we remove the Time or Priority label, making it impossible for a bounty hunter to pick up

ok

**Reply by 0x4007** [reply]
If you think all the points are addressed in your original discussion topic, consider closing it as completed.
